### PR TITLE
update nextcloud-tor.md (fixes #809)

### LINKED ISSUE #809: Grammar Error/Awkward Syntax for Steps to SSH Into Pi
### Problem
The instructions to ssh into the Raspberry Pi in Step 7 were grammatically incorrect or had awkward syntax that made it confusing to follow:

> Open up your terminal or command prompt, and run `ssh root@[IP address]` to enter the root of your Pi or, to log into the Pi user use `ssh pi@[local IP address]` and to log into root, you can run `sudo -s`.

 ### Steps to reproduce the problem
N/A

 ### Screenshots
image

 ### Proposed solution
Revise the text to read:

> Use one of the following ways to enter the root of your Pi:
> 
> 1. Open up your terminal or command prompt, and run `ssh root@[IP address]`.
> 2. First log into the Pi user by using `ssh pi@[local IP address]`, then log into the root by running `sudo -s`.

### PR DESCRIPTION
**Pull request by JLKwong**
Fixes #809 

### Description
**DELETED**
> Open up your terminal or command prompt, and run `ssh root@[IP address]` to enter the root of your Pi or, to log into the Pi user use `ssh pi@[local IP address]` and to log into root, you can run `sudo -s`.

**ADDED**
> Use one of the following ways to enter the root of your Pi:
> 
> 1. Open up your terminal or command prompt, and run `ssh root@[IP address]`.
> 2. First log into the Pi user by using `ssh pi@[local IP address]`, then log into the root by running `sudo -s`.

### Raw.Githack preview link
https://raw.githack.com/JLKwong/JLKwong.github.io/809-Grammar-Fix-SSH-Pi-Step-7/#!pages/vi/nextcloud-tor.md

### CODE REVIEW

File: pages/vi/nextcloud-tor.md [hunk 1]
```diff
@@ -61,7 +61,11 @@ Install [Tor](https://www.torproject.org/download/)
 
 Plug the microSD into your Raspberry Pi, and power it on.  The red LED should turn on, indicating that the Raspberry Pi is connected to power.  Once the green LED next to it on your Raspberry Pi stabilizes into a solid green, you should see "treehouses" appear in available Wifi networks.  Connect to it, and make sure that you have an internet connection by opening up another web page.  
 
-Find the Raspberry Pi's IP address in your network settings (it will probably be `192.168.2.1`).  Open up your terminal or command prompt, and run `ssh root@[IP address]` to enter the root of your Pi or, to log into the Pi user use `ssh pi@[local IP address]` and to log into root, you can run `sudo -s`.
+Find the Raspberry Pi's IP address in your network settings (it will probably be `192.168.2.1`). Use one of the following ways to enter the root of your Pi:
+ 
+1. Open up your terminal or command prompt, and run `ssh root@[IP address]`.
+2. First log into the Pi user by using `ssh pi@[local IP address]`, then log into the root by running `sudo -s`.
```
Comment (rjpadilla) [inline pages/vi/nextcloud-tor.md]: This looks fine, but you may want to consider changing the list to unordered. The way its formatted looks like to me you're giving me instructions to follow in a sequential order.

Comment (BryanGazali) [reply pages/vi/nextcloud-tor.md]: I second this comment

**Review by BryanGazali — DISMISSED** [review]

File: pages/vi/nextcloud-tor.md (hunk 1 above)
Comment (JLKwong) [reply pages/vi/nextcloud-tor.md]: Oof, thanks for the feedback. I changed the commit to use an unordered list instead. 😌👍

**Comment by JLKwong** [comment]
Per both of your feedback, I pushed another commit to add this text instead:
> Find the Raspberry Pi's IP address in your network settings (it will probably be 192.168.2.1). Open up your terminal or command prompt, and use one of the following ways to enter the root of your Pi:
>  * run ssh root@[IP address] to enter the root of your Pi using the password (root password).
>  * First log into the Pi user by using ssh pi@[local IP address] using the (default) password "raspberry", then log into the root by running sudo -s.

**Review by dogi — APPROVED** [review]

**Review by BryanGazali — APPROVED** [review]
Good

**Pull request merged**